material-overrides/main.html

```diff
@@ -195,63 +195,102 @@
     });
   </script>
   <script>
+    // AI_RAW_BASE = base URL where AI artifacts are stored 
+    const AI_RAW_BASE = "https://raw.githubusercontent.com/polkadot-developers/polkadot-docs/master/.ai'";
```

Comment:
```suggestion
    const AI_RAW_BASE = "https://raw.githubusercontent.com/polkadot-developers/polkadot-docs/master/.ai";
```

single quote was causing issues